Riot/Modules/Room/RoomViewController.m

```diff
@@ -7260,6 +7116,7 @@ - (void)updateThreadListBarButtonBadgeWith:(MXThreadingService *)service
 - (void)updateThreadListBarButtonItem:(UIBarButtonItem *)barButtonItem with:(MXThreadingService *)service
 {
     // Tchap: Threads are disabled in Tchap
+//    if (!service)
```

Comment:
Where coming from this change ?

Reply:
I think I made an error by changing this in the previous merge (or it was in Element code, but I don't think so).
So I revert to the current Element code, and added the next line.
But I also can comment those lines like the others instead.